src/main/java/de/tum/in/www1/artemis/web/rest/dto/competency/LearningPathHealthDTO.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.in.www1.artemis.web.rest.dto.competency;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public record LearningPathHealthDTO(@NotNull HealthStatus status, Long missingLearningPaths) {
```

Comment:
will the second parameter ever be null?

Reply:
This will only be true if the HealthStatus is `MISSING`. Otherwise, this is irrelevant information (`OK`~0 missing, `DISABLED`~all are missing, irrelevant since LPs are disabled and would require the server to retrieve the number of enrolled students).